doc/api/deprecations.md

```diff
@@ -2136,7 +2136,7 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Documentation-only (supports [`--pending-deprecation`][])
```

Comment:
I think this change deserves to be included in the metadata section? :)

Reply:
@tniessen yup, done!